ramls/traits/filterable.raml

```diff
@@ -2,13 +2,15 @@ queryParameters:
   filter[selected]:
     displayName: Selection status
     type: string
-    enum: ["all", "true", "false", "ebsco"]
     description: Filter to narrow down results based on selection status
+    example: |
+      ["all", "true", "false", "ebsco"]
     required: false
   filter[type]:
     displayName: Content type
     type: string
-    enum: ["all", "aggregatefulltext", "abstractandindex", "ebook", "ejournal", "print", "unknown", "onlinereference"]
```

Comment:
why is this enum removed?

Reply:
because of the issue which was described by Andrii Piais above.
If we leave values as enum then:
> RestVerticle then compares them with query parameters using case senstive comparison with value returned from toString(), so query parameters will be parsed correctly only if we send this in query "?sort=RELEVANCE",

Reply:
Gotcha! Makes sense then to not have enum there and have them in examples like Andrii is doing in his PR